web/packages/teleport/src/features.tsx

```diff
@@ -99,31 +98,6 @@ class AccessRequests implements TeleportFeature {
   topMenuItem = this.navigationItem;
 }
 
-export class FeatureNodes implements TeleportFeature {
-  route = {
-    title: 'Servers',
-    path: cfg.routes.nodes,
-    exact: true,
-    component: Nodes,
-  };
-
-  navigationItem = {
-    title: NavTitle.Servers,
-    icon: Server,
-    exact: true,
-    getLink(clusterId: string) {
-      return cfg.getNodesRoute(clusterId);
-    },
-  };
-
-  category = NavigationCategory.Resources;
-
-  hasAccess(flags: FeatureFlags) {
-    return flags.nodes;
-  }
-}
-
-// TODO (avatus) add navigationItem when ready to release
```

Comment:
Thank you 🙏

Reply:
Wow, at last! I have to admit I walked past it a couple of times, every time saying "meh, not today".